examples/documentation/PTI_experiment/PTI_Experiment_Recon3D_anisotropic_target_small.py

```diff
@@ -0,0 +1,695 @@
+)
+phase_PT, absorption_PT, mean_permittivity_PT = [
+    optics.unit_conversion_from_scattering_potential_to_permittivity(
+        SP_array, lambda_illu, n_media=n_media, imaging_mode="3D"
+    )
+    for SP_array in [phase, f_tensor[1].copy(), mean_permittivity]
+]
+mean_permittivity_PT = np.array(
+    [
+        ((-1) ** i)
+        * util.wavelet_softThreshold(
+            ((-1) ** i) * mean_permittivity_PT[i], "db8", 0.00303, level=1
+        )
+        for i in range(2)
+    ]
+)
+
+
+# %%
+# Visualize reconstructed physical properties of the anisotropic glass target
+
+# %%
+# Reconstructed phase, absorption, mean permittivity, azimuth, and inclination assuming (+) and (-) optic sign
+
+# %%
+# browse the reconstructed physical properties
+visual.parallel_4D_viewer(
+    np.transpose(
+        np.stack(
+            [
+                phase_PT,
+                mean_permittivity_PT[0],
+                azimuth[0],
+                theta[0],
+                absorption_PT,
+                mean_permittivity_PT[1],
+                azimuth[1],
+                theta[1],
+            ]
+        ),
+        (3, 0, 1, 2),
+    ),
+    num_col=4,
+    origin="lower",
+    set_title=True,
+    titles=[
+        r"phase",
+        r"mean permittivity (+)",
+        r"$\omega$ (+)",
+        r"$\theta$ (+)",
+        r"absorption",
+        r"mean permittivity (-)",
+        r"$\omega$ (-)",
+        r"$\theta$ (-)",
+    ],
+)
+
+# %%
+# (Obsolete, not maintained) save results to zarr array
+
+# writer = WaveorderWriter('.', hcs=False, hcs_meta=None, verbose=True)
+# writer.create_zarr_root('Anisotropic_target_small_processed.zarr')
+# chan_names = ['f_tensor0r', 'f_tensor0i', 'f_tensor1c','f_tensor1s','f_tensor2c','f_tensor2s', 'f_tensor3', 'mat_map0', 'mat_map1']
+# PTI_array = np.transpose(np.concatenate((f_tensor, mat_map),axis=0)[np.newaxis,...],(0,1,4,2,3)) # dimension (T, C, Z, Y, X)
+# data_shape = PTI_array.shape
+# chunk_size = (1,1,1)+PTI_array.shape[3:]
+# writer.init_array(0, data_shape, chunk_size, chan_names, position_name='f_tensor', overwrite=True)
+# writer.write(PTI_array, p=0)
+
+# chan_names_phys = ['Phase3D', 'Retardance3D', 'Orientation', 'Inclination', 'Optic_sign']
+# phys_data_array = np.transpose(np.array([phase_PT, np.abs(retardance_pr_PT[1]), azimuth[1], theta[1], p_mat_map]),(0,3,1,2))[np.newaxis,...]
+# data_shape_phys = phys_data_array.shape
+# dtype = 'float32'
+# writer.init_array(1, data_shape_phys, chunk_size, chan_names_phys, dtype, position_name='Stitched_physical', overwrite=True)
+# writer.write(phys_data_array, p=1)
+
+# %%
+# Load the processed results
+PTI_file_name = (
+    "/path/to/Anisotropic_target_small/Anisotropic_target_small_processed.zarr"
+)
+reader = zarr.open(PTI_file_name, mode="r")
+PTI_array = np.array(reader["Row_0/Col_0/f_tensor/array"])
+print(PTI_array.shape)
+PTI_array = np.transpose(PTI_array, (0, 1, 3, 4, 2))[0]
+f_tensor = PTI_array[:7]
+mat_map = PTI_array[7:]
+
```

Comment:
Can we assign names to these channels? I understand the previous version of iohub also had an API for that.